src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc

```diff
@@ -0,0 +1,34 @@
+= Hawkular Alerting: Tutorial Now Available!
+Jay Shaughnessy
+2017-03-14
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerting
+
+Hawkular Alerting is a pluggable, API-driven component for adding alerting to your application or tool.  It's one of the many http://hawkular.org[Hawkular] offerings for monitoring and management.  To help quick-start new users, and as a reference for current users, we've released the first five lessons of a new tutorial, with more to follow.  It's a great way to hit the ground running.
+
+====== For Git Users:
+
+----
+git clone https://github.com/hawkular/hawkular-alerts.git
+cd hawkular-alerts/examples/tutorial
+----
+
+====== For Others:
+
+. Click this link to visit http://github.com/hawkular/hawkular-alerts[Hawkular Alerting on GitHub].
+. Click _Clone or download_
+. Click _Download ZIP_
+
```

Comment:
OK, so using the larger section heading did the trick.  Glad you found happiness :)